src/models/launch_lite_1/README.md

```diff
@@ -0,0 +1,44 @@
+# Launch Lite (launch_lite_1)
+
+- [External Overview](./external-overview.md)
+- [Customization & Repairs](./repairs.md)
+
+![Launch Lite Photo](./img/launch-lite.webp)
+
+The System76 Launch Lite is a configurable keyboard with the following specifications:
+
+- Chassis
+  - [Open-source milled chassis design](https://github.com/system76/launch/tree/master/chassis/launch-lite)
+  - Magnetically-attachable 15-degree angle lift bar
+- Electronics
+  - [Open-source PCB design](https://github.com/system76/launch/tree/master/pcb-lite)
+  - Individually addressable RGB LED backlighting
+  - N-key rollover
+- Sockets and Switches
+  - Kailh MX hot swap sockets
+  - Default switch options:
+    - NovelKeys x Kailh Box Jade
```

Comment:
As on the Launch, I think we should follow the sales description:
 * Kailh Box Jade - Clicky
 * Kailh Box royal - Tactile
 * Kailh Box Silent Pink - Silent Linear
 * Kailh Box Silent Brown - Silent